src/signals/incident/definitions/wizard-step-2-vulaan/afval-container.ts

```diff
@@ -19,6 +19,7 @@ export const controls = {
     meta: {
       ifOneOf: {
         subcategory: [
+          'container-bijplaatsing',
```

Comment:
This change shouldn't be necessary, since the logic that loads these controls already looks if a subcategory start with the value `container`.

Reply:
In fact, the whole `ifOneOf` should be removable.

Reply:
I wasn't sure if there was a reason for the double check. I will remove the part that checks the ifOneOf array then